workspace/apps/pidp/src/scss/material/_material-palette.scss

```diff
@@ -38,13 +37,13 @@ $bcgov-blue: (
 );
 
 // http://mcg.mbitson.com/#!?mcgpalette0=%23fcba19
-$bcgov-yellow: (
+$material-theme-secondary: (
   50: #fff7e3,
   100: #feeaba,
   200: #fedd8c,
   300: #fdcf5e,
   400: #fcc43c,
-  // BC Gov. Secondary
+  // Material Theme Secondary
```

Comment:
You can remove this comment.  It was only there when we were paired with BC Gov theming.